randomize/randomize.go

```diff
@@ -60,6 +60,7 @@ func Struct(s *Seed, str interface{}, colTypes map[string]string, canBeNull bool
 	copyBlacklist := make([]string, len(blacklist))
 	copy(copyBlacklist, blacklist)
 	blacklist = copyBlacklist
+	blacklist = append(blacklist, "deleted_at")
```

Comment:
This should go in the calls to randomize, not in the randomize function itself I think.

Reply:
I understand that adding in the calls to randomize is neater. However, I found that there are plenty of places (~50) that need to be modified and the logic needs to be duplicated everywhere. Therefore I ended up changing the `randomize.Struct` func. The only potential issue I can think of is that the `deleted_at` column serves some other purposes and can't be null, but I'm not sure if it's a valid concern and I decided to go to this conventional approach.

Reply:
Okay. I'll probably merge as-is and then attempt to do some fancy regex replaces to put this in the correct spot.